02.calendar/calendar.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+/*入力の有無によって値を変更*/
```

Comment:
不要なコメントかも？
[【動画付き・初心者向け】コメントにはWHYを書こう 〜不要なコメントと必要なコメントの違いについて〜 \#Ruby \- Qiita](https://qiita.com/jnchito/items/f0d90af4ed44b7484103)

仮にコメントするとしても

```suggestion
// 入力の有無によって値を変更
```

とすることが多いですね

Reply:
不要なコメントでした、申し訳ございません。
必要最低限のコメントのみにしました。
[📃 chore: 不要なコメントを削除](https://github.com/reckyy/js-practices/pull/2/commits/7c7b78384d5ee77f9ae856aec44df50f6535a47f)